jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -788,7 +791,7 @@ public Action process() throws Exception
                         if (_head || _generator.isNoContent())
                         {
                             BufferUtil.clear(_chunk);
-                            BufferUtil.clear(_content);
+                            Arrays.stream(_content).forEach(BufferUtil::clear);
```

Comment:
I always worry about stream API on critical path code..... is this going to allocate an iterator when a simple for loop would not?  @lorban @sbordet ???